collectors/performance/appdynamics/src/main/java/com/capitalone/dashboard/collector/DefaultAppdynamicsClient.java

```diff
@@ -160,7 +161,11 @@ public static String joinURL(String base, String... paths) throws MalformedURLEx
                 for (Object entry : array) {
                     JSONObject jsonEntry = (JSONObject) entry;
                     String metricPath = getString(jsonEntry, "metricPath");
-                    JSONObject mObj = (JSONObject) getJsonArray(jsonEntry, "metricValues").get(0);
+                    JSONArray metricValues =  getJsonArray(jsonEntry, "metricValues");
+                    if (metricValues.isEmpty()){
```

Comment:
Can you please replace with CollectionUtils.isEmpty() here?

Reply:
I try to change, but the type of this object (JSONArray) is not compatible with this function

Reply:
This is the reason because I try to use a similar function